DTS-STN/Alpha-Site
# Analytics implementation

# Description

[Go through the Adobe Analytics Implementation Testing Process manual alongside the Implementation Run Testing Checklists](https://trello.com/c/Ov9CbyGo/287-go-through-the-adobe-analytics-implementation-testing-process-manual-alongside-the-implementation-run-testing-checklists)

I completed the form tracking with the submit buttons.

## Acceptance Criteria

The submit button on the report a problem dropdown and the submit form should have a data-gc-analytics-submit attribute.

## Test Instructions

1. Go on the signup page and inspect the submit button, it should have a data-gc-analytics-submit attribute
2. The report a problem's submit button should also have a data-gc-analytics-submit attribute

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
